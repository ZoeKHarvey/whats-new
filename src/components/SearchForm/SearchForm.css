.search-form {
  position: absolute;
  height: 12vh;
  display: flex;
  width: 75%;
  margin-left: 17%;
  margin-top: 0%;
  display: flex;
  justify-content: space-between;
  background-color: #141527;
}

input,
.button__search {
  padding: -3%;
  height: 55%;
  margin-top: 2%;
  border: none;
  font-size: 77px;
}

input {
  width: 50%;
  background-color: #141527;
}

.button__search {
  border-radius: 125px;
  width: 20%;
  background-color: #B56AF0;
  color: white;
}