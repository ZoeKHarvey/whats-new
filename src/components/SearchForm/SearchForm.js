import React, { Component } from 'react';
import './SearchForm.css';

// class SearchForm extends Component {
//   constructor() {
//     super();


//   }



//   render() {
//     return (
//       <form>
//         <input
//           type="text"
//           placeholder="Search for local news articles here."
//         />
  
//         <button>Search Now</button>
//       </form>
//     )
//   }

// }


export default SearchForm;