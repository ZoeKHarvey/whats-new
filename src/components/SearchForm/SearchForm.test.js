  
import SearchForm from './SearchForm';
import React from 'react';
import { shallow } from 'enzyme';

describe('SearchForm', () => {
  const searched = jest.fn();

  it('should match the snapshot information passed correctly', () => {
    const wrapper = shallow(
      <SearchForm
        searched={searched}
      />
    )
    expect(wrapper).toMatchSnapshot();
  })

})