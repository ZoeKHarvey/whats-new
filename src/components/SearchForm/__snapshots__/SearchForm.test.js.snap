// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm should match the snapshot information passed correctly 1`] = `
<div
  className="search-form"
>
  <input
    onChange={[Function]}
    placeholder="Search local news here."
    type="text"
    value=""
  />
  <button
    className="button__search"
  >
    Search Now
  </button>
</div>
`;
